Packs/Remedy-On-Demand/Integrations/RemedyOnDemand/RemedyOnDemand.yml

```diff
@@ -37,23 +35,24 @@ configuration:
   name: incidentType
   type: 13
   required: false
+- defaultvalue: '1'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
```

Comment:
Same question as the previous PR, why did you add this param? it should be added automatically when adding the `isfetch: true` key with a default value of 1.

Reply:
I answered this question in the previous PR: "I did not add this param, seems like XSOAR automatically added it? I see this param is in the existing Remedy integration already, why does it show as being added?"
I can remove this param since it should be added automatically when adding the `isfetch: true` key

Reply:
I see you already removed it, thx!